build/pipeline-ci.yaml

```diff
@@ -11,6 +11,7 @@ schedules:
   branches:
```

Comment:
Is there an additional benefit to running the CI build nightly? Should it be enough to run just the release build nightly, since it has everything?

Reply:
We used to have a reason before i made the updates to the pipeline as they were slightly different but now that they are so similar it should be fine to only run the release nightly and run the CI on merges to master